### DISCUSSION TITLE
how can i become active contributor

### DISCUSSION THREAD

**Discussion by YashRajSingh-4799** [community/community#49934] [GitHub Education]
![image](https://user-images.githubusercontent.com/64195098/224750692-33e8fe79-340d-49df-95ba-bcef54b46849.png)

will it count personal project commits also or i just have to make contribution in organizations repo

**Comment by maahaMaya** [comment]
Both 
1. Create own project and contribute
2. Or else contribute in another project

**Comment by Saifuddin53** [comment]
at least in what period I can become an active contributor ?

**Reply by maahaMaya** [reply]
it depends on your commit and days of commit in github.

**Comment by Ntshangase** [comment]
Asking this question is a great start. You can also check out: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile  To see how GitHub counts contributions.
